src/Bookings.js

```diff
@@ -12,7 +12,7 @@ const Bookings = () => {
     <div className="App-content">
       <div className="container">
         <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+        <SearchResults results={FakeBookings} />
```

Comment:
For Task 14, you need to set up a new state first and then pass that into `SearchResults`. If you are having troubles with this task, please book in time with me.